components/generator/imagevector/src/test/resources/kt/lazy/WithoutPath.indent4.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.composegears.valkyrie.icons
+
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.dp
+
+val WithoutPath: ImageVector by lazy(LazyThreadSafetyMode.NONE) {
```

Comment:
`WithoutPath.indent4.kt` equal to `WithoutPath.kt`

maybe we can add test to indent size 6?
The same for icon pack

Reply:
make sense